crates/bevy_diagnostic/src/system_information_diagnostics_plugin.rs

```diff
@@ -32,6 +32,10 @@ impl SystemInformationDiagnosticsPlugin {
     pub const CPU_USAGE: DiagnosticPath = DiagnosticPath::const_new("system/cpu_usage");
     /// Total system memory usage in %
     pub const MEM_USAGE: DiagnosticPath = DiagnosticPath::const_new("system/mem_usage");
+    /// Process cpu usage in %
+    pub const PROCESS_CPU_USAGE: DiagnosticPath = DiagnosticPath::const_new("process/cpu_usage");
```

Comment:
I feel like the other ones should probably be renamed to `GLOBAL_CPU_USAGE` or `SYSTEM_CPU_USAGE` to make the difference clearer. (same for mem of course)